compute/admin_guide/tools/twistcli.adoc

```diff
@@ -113,103 +113,153 @@ Collects log data from a node and uploads it to Prisma Cloud's support area.
 The _twistcli_ tool offers feature parity across all supported operating systems, with a few exceptions.
 The following table highlights where functions are disabled, or work differently, on a given platform.
 
-[cols=".^2,2,1,1,1", frame="topbot"]
+[cols=".^2,2,1,1,1,1,1", frame="topbot"]
 |====
-2+^| twistcli 3+^| Platform
+2+^| twistcli 5+^| Platform
 
 |Command
 |Subcommand
 |Linux
+|Linux ARM64
 |macOS
+|macOS ARM64
 |Windows
 
 .3+|`console` {set:cellbgcolor:#fff}
 |`export`
 |Yes
+|No
+|Yes
 |Yes
 |Yes
 
 |`install`
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# {set:cellbgcolor:#000}
+|No
+|No
+|No
+|No
 
 |`uninstall` {set:cellbgcolor:#fff}
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No# {set:cellbgcolor:#000}
+|Yes
+|No
+|No
+|No
 
 .3+|`defender` {set:cellbgcolor:#fff}
 |`export`
 |Yes
 |Yes
 |Yes
+|Yes
+|Yes
 
 |`install`
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No#
+|Yes
+|No
+|No
+|No
 
 |`uninstall` {set:cellbgcolor:#fff}
 |Yes
-|[white]#No# {set:cellbgcolor:#000}
-|[white]#No#
+|No
+|No
+|No
+|No
 
 |`hosts` {set:cellbgcolor:#fff}
 |`scan`
 |Yes
-|[white]#No^1^# {set:cellbgcolor:#000}
-|[white]#No#
+|Yes
+|No^1^
+|No
+|No
 
 |`images` {set:cellbgcolor:#fff}
 |`scan`
 |Yes
+|Yes
 |Yes^2^
+|Yes
 |Yes^3^
 
 .2+|`intelligence`
 |`upload`
 |Yes
 |Yes
```

Comment:
@ssugandh let's have this as "No" as well, as we are not sure and didn't test it

Reply:
@grevach : Do you mean No for Linux?